src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -54,7 +54,12 @@ foam.CLASS({
     {
       class: 'String',
       name: 'browseTitle',
-      expression: function(of) { return foam.String.pluralize(of.model_.label); }
+      expression: function(of) { return  of.model_.plural }
```

Comment:
remove extra space before 'of'. Add ; after plural.

Reply:
will push to this https://github.com/Nauna/foam2/pull/82